com.unity.ml-agents/Runtime/Policies/BrainParameters.cs

```diff
@@ -51,7 +53,12 @@ public class BrainParameters
         [Range(1, 50)] public int NumStackedVectorObservations = 1;
 
         /// <summary>
-        /// The size of the action space.
+        /// The specification of the Action space for the BrainParameters.
+        /// </summary>
+        public ActionSpec VectorActionSpec = new ActionSpec(0, 0, new int[] { });
```

Comment:
Can we just call this "ActionSpec"? https://unity.slack.com/archives/C01G1362C23/p1607471246002900

Reply:
I kept this simply because `public ActionSpec ActionSpec = new ActionSpec();` looks a bit confusing.

Reply:
Is it okay to add new public fields to BrainParameters?

Reply:
Yes, you can always add new public fields and methods.

Reply:
We do it in a few other places like this:
https://github.com/Unity-Technologies/ml-agents/blob/e3990b578e154970377b57bcffaf90c023b9589b/com.unity.ml-agents/Runtime/Actuators/ActuatorComponent.cs#L22
and 
https://github.com/Unity-Technologies/ml-agents/blob/e3990b578e154970377b57bcffaf90c023b9589b/com.unity.ml-agents/Runtime/Actuators/VectorActuator.cs#L75

I think it's better to have a slightly confusing declaration and simplify how people use it.